dashboard/views/_committee.py

```diff
@@ -98,51 +96,44 @@ def committee_event(request, event_id):
         'form': form,
     }
 
-    return render(request, "committee_event.html", context)
+    return render(request, "committee-event.html", context)
 
 
-@verify_position(['Public Relations Chair', 'Scholarship Chair', 'Service Chair', 'Philanthropy Chair', 'Alumni Relations Chair', 'Membership Development Chair', 'Social Chair', 'Vice President of Health and Safety', ' Recruitment Chair', 'Vice President', 'President', 'Adviser'])
-def committee_event_add(request, position):
+@verify_position(['pr-chair', 'scholarship-chair', 'service-chair', 'philanthropy-chair', 'alumni-relations-chair', 'memdev-chair', 'social-chair', 'vphs', 'recruitment-chair', 'vice-president', 'president', 'adviser'])
```

Comment:
Can we use the `Position.PositionChoices` here.  I do realize this kind of means we should change this everywhere and thats probably gonna take a lil bit, so I'm cool with merging this without making the changes to `verify_position` calls, but we should make an issue to do this as it would be good to do